09-Time-Series-Data.ipynb

```diff
@@ -0,0 +1,786 @@
+    "\n",
+    "* GOES XRS LightCurve\n",
+    "* SDO EVE LightCurve for level 0CS data\n",
+    "* Proba-2 LYRA LightCurve\n",
+    "* NOAA Solar Cycle monthly indices.\n",
+    "* Nobeyama Radioheliograph Correlation LightCurve.\n",
+    "* RHESSI X-ray Summary LightCurve.\n",
+    "\n",
+    "We're going to examine the lightcurves created by a solar flare on June 7th 2011.\n",
+    "\n",
+    "Lets begin with the import statements:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function, division\n",
+    "import numpy as np\n",
+    "import sunpy\n",
+    "from sunpy import lightcurve as lc\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now lets create some lightcurves"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "goes_lightcurve = lc.GOESLightCurve.create('2011-06-07 06:00','2011-06-07 08:00')\n",
+    "\n",
+    "hsi_lightcurve = lc.RHESSISummaryLightCurve.create('2011-06-07 06:00','2011-06-07 08:00')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In terms of LYRA, the server only allows you to download an entire day of data at a time. We can match this to the rest of the daya by using the truncate function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "lyra_lightcurve_fullday = lc.LYRALightCurve.create('2011-06-07')\n",
+    "lyra_lightcurve = lyra_lightcurve_fullday.truncate('2011-06-07 06:00','2011-06-07 08:00')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Part of the advantage of using these inbuilt functions we can get a quicklook at our data using simple commands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fig = goes_lightcurve.peek()\n",
+    "fig = lyra_lightcurve.peek()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "### Accessing and using the data\n",
+    "\n",
+    "More custom plots can be made easily by accessing the data in the lightcurve object. Both the time information and the data are contained within the lightcurve.data object, which is a pandas dataframe. We can see what data is contained in the dataframe by finding which keys it contains and also asking what's in the meta data: "
```

Comment:
This is fine, but probably try and avoid saying object too much, since we haven't explained what objects are.